pkgs/by-name/sd/SDL_audiolib/package.nix

```
@@ -29,10 +30,28 @@ stdenv.mkDerivation (finalAttrs: {
   buildInputs = [
     SDL2
     flac
+  ]
+  ++ lib.optionals stdenv.isDarwin [
+    fmt
   ];
 
   strictDeps = true;
 
+  # pre-packaged fmt library not working on darwin
+  postPatch = lib.optionalString stdenv.isDarwin ''
+        # Remove the bundled fmt directory
+        rm -rf 3rdparty/fmt
+
+        # Patch CMakeLists.txt to skip adding bundled fmt when USE_SYSTEM_FMT is true
+        substituteInPlace CMakeLists.txt \
+          --replace 'add_bundled_fmtlib()' 'if(NOT USE_SYSTEM_FMT)
+        add_bundled_fmtlib()
+    endif()
+    if(USE_SYSTEM_FMT)
+        find_package(fmt REQUIRED)
+    endif()'
```

There's no point in making it configurable.
```suggestion
          --replace-fail 'add_bundled_fmtlib()' 'find_package(fmt REQUIRED)'
```